package.json

```diff
@@ -47,15 +45,14 @@
     "express": "~4.13.1",
     "express-session": "^1.13.0",
     "isomorphic-fetch": "^2.2.1",
-    "mongoose": "^4.5.0",
-    "nodemon": "^1.9.2",
```

Comment:
we will need nodemon when we deploy using a docker container, this was used during the demo